vanhouse/src/components/WelcomeUser.js

```diff
@@ -20,24 +20,25 @@ function WelcomeUser({
         }).then((response) => {
             response.json()
                 .then((resJSON => {
-                    console.log(resJSON);
-                    // localStorage.setItem("currentUser", JSON.stringify(resJSON));
                     setUser({
                         userId: resJSON.userId,
                         email: resJSON.email,
                         firstName: resJSON.firstName,
-                        lastName: resJSON.lastName
+                        lastName: resJSON.lastName,
+                        admin: resJSON.admin
                     });
-                }));
-        }).catch(err => {
-            console.log("Not logged in");
+                }))
+                .catch(() => {
+                    setUser(null);
+                })
+        }).catch(() => {
             setUser(null);
         });
     }
 
     useEffect(() => {
         loggedInCondRender();
-    }, []);
+    });
```

Comment:
@parksj23 I did some experiments and I think the bug is here. If you change it to
```
useEffect(() => {
        loggedInCondRender();
    },[]);
```
Then it works well. Without the dependency, it will be stuck in a re-render loop.

Reply:
I think I'm noticing that too. Other than that it looks good!

Reply:
Oh yes, I did notice it but I was trying to get it to constantly check if the jwt cookie was still there, so the greeting would disappear as soon as the jwt cookie expired. I will change it back though because I agree it sends too many requests. Then we'd just have to refresh after the cookie disappears to get rid of the greeting. Thank you!!